ansible/pbTestScripts/testScript.sh

```diff
@@ -82,70 +85,71 @@ checkVars()
 
 checkVagrantOS()
 {
-	case "$vagrantOS" in
-		"Ubuntu1604" | "U16" | "u16" )
-			vagrantOS="Ubuntu1604";;
-		"Ubuntu1804" | "U18" | "u18" )
-			vagrantOS="Ubuntu1804";;
-		"CentOS6" | "centos6" | "C6" | "c6" )
-			vagrantOS="CentOS6" ;;
-		"CentOS7" | "centos7" | "C7" | "c7" )
-			vagrantOS="CentOS7" ;;
-		"Windows2012" | "Win2012" | "W12" | "w12" )
-			vagrantOS="Win2012";;
-		"all" )
-			vagrantOS="Ubuntu1604 Ubuntu1804 CentOS6 CentOS7 Windows2012" ;;
-		*) echo "Not a currently supported OS" ; vagrantOSList; exit 1;
-	esac
+	cd ${WORKSPACE}/adoptopenjdkPBTests/${folderName}-${branchName}/ansible
+	local vagrantOSlist=$(ls -l Vagrantfile.* | cut -d. -f 2)
```

Comment:
That needs to be `-1` not `-l` otherwise the list will be gibberish

Reply:
In my testing, both output the same as `cut`s delimiter is `.` However, I do agree that `-1` would be better :-)

Reply:
Not sure where you've done your testing - on a CentOS6 VM there's a `.` character after the permissions field at the start:
```
[vagrant@adoptopenjdkC6A vagrant]$ ls -l Vagrantfile.*
-rw-r--r--. 1 vagrant vagrant 1872 Dec 16 03:51 Vagrantfile.CentOS6
-rw-r--r--. 1 vagrant vagrant 1864 Dec  2 10:21 Vagrantfile.CentOS6~
-rw-r--r--. 1 vagrant vagrant 1864 Dec  2 10:21 Vagrantfile.CentOS7
-rw-r--r--. 1 vagrant vagrant 1963 Dec  2 10:21 Vagrantfile.Ubuntu1604
-rw-r--r--. 1 vagrant vagrant 1963 Dec  2 10:21 Vagrantfile.Ubuntu1804
-rw-r--r--. 1 vagrant vagrant 1925 Dec  2 10:21 Vagrantfile.Win2012
[vagrant@adoptopenjdkC6A vagrant]$ 
```

Reply:
> Not sure where you've done your testing
Not on a CentOS6 VM ;-) This will only be executed on the Jenkins machine, which I believe will only ever be Ubuntu - and that doesn't have a dot. Output from the `infra-vagrant-1` machine : 
```
lrwxrwxrwx  1 jenkins jenkins   22 Dec  9 09:05 Vagrantfile -> Vagrantfile.Ubuntu1604
-rw-rw-r--  1 jenkins jenkins 2208 Dec  9 09:05 Vagrantfile.CentOS6
-rw-rw-r--  1 jenkins jenkins 2208 Dec  9 09:05 Vagrantfile.CentOS7
-rw-rw-r--  1 jenkins jenkins 2052 Dec  9 09:05 Vagrantfile.Debian8
-rw-rw-r--  1 jenkins jenkins 2307 Dec  9 09:05 Vagrantfile.Ubuntu1604
-rw-rw-r--  1 jenkins jenkins 2307 Dec  9 09:05 Vagrantfile.Ubuntu1804
-rw-rw-r--  1 jenkins jenkins 1965 Dec  9 09:05 Vagrantfile.Win2012
```
Regardless, it's been changed 👍